tests/e2e/src/pages/base.page.ts

```diff
@@ -47,6 +47,23 @@ export class BasePage {
     await this.world.page?.fill(selector, text, config.increasedTimeout);
   }
 
+  async fillSpecificField(inputIDType: string, inputIDName: string, text: string) {
+    const helper = new Helper(this.world);
+    selector = `//*[@${inputIDType}="${inputIDName}"]`;
+    console.log(await this.world.page?.locator(selector).first().isVisible());
+    if (await this.world.page?.locator(selector).first()) {
```

Comment:
Not an issue? await this.world.page?.locator(selector).first() --> Visible or just first? 
in case of Visible it is better to use an existing method: checkElementVisible() from helper.ts